addons/rating/models/rating_mixin.py

```diff
@@ -192,7 +192,7 @@ def rating_apply(self, rate, token=None, feedback=None, subtype_xmlid=None):
             rating.write({'rating': rate, 'feedback': feedback, 'consumed': True})
             if hasattr(self, 'message_post'):
                 feedback = tools.plaintext2html(feedback or '')
-                self.message_post(
+                self.with_user(rating.create_uid).sudo().message_post(
```

Comment:
:thinking:

Reply:
@tde-banana-odoo 
Here we want to add rating user as the author of msg.
for example, data and test cases will be run by the odoobot, and in that case, the author will be odoobot.

Reply:
I think the concern is because this is probably not the right place to do it, this should be done much higher in the stack, during the creation of demo/data.

Reply:
@seb-odoo @tde-banana-odoo 
Changes are done. can you please check it?
Thanks